packages/react-query/src/__tests__/useInfiniteQuery.test.tsx

```diff
@@ -71,7 +71,7 @@ describe('useInfiniteQuery', () => {
     function Page() {
       const state = useInfiniteQuery({
         queryKey: key,
-        queryFn: ({ pageParam }) => Number(pageParam),
+        queryFn: ({ pageParam }) => sleep(10).then(() => Number(pageParam)),
```

Comment:
```suggestion
        queryFn: ({ pageParam }) => sleep(10).then(() => pageParam),
```

This seems to be a leftover from TanStack Query v4! Now, TanStack Query v5 has improved this type of thing and I think this part should be improved incrementally throughout the entire code.
We can remove unnecessary code like this. I would appreciate it if you could take care of this too!

Reply:
> This seems to be a leftover from TanStack Query v4! Now, TanStack Query v5 has improved this type of thing and I think this part should be improved incrementally throughout the entire code. We can remove unnecessary code like this. I would appreciate it if you could take care of this gradually too!

@manudeli 

I totally agree with you. I’ll take care of this part as well in future tasks.